src/solver/optim-model-filler/ComponentFiller.cpp

```diff
@@ -0,0 +1,57 @@
+
+#include "antares/solver/optim-model-filler/ComponentFiller.h"
+
+// TODO: remove this unnecessary includes
+#include "antares/solver/expressions/nodes/ComparisonNode.h"
+#include "antares/solver/expressions/nodes/ComponentNode.h"
+#include "antares/solver/expressions/nodes/DivisionNode.h"
+#include "antares/solver/expressions/nodes/EqualNode.h"
+#include "antares/solver/expressions/nodes/GreaterThanOrEqualNode.h"
+#include "antares/solver/expressions/nodes/LessThanOrEqualNode.h"
+#include "antares/solver/expressions/nodes/LiteralNode.h"
+#include "antares/solver/expressions/nodes/MultiplicationNode.h"
+#include "antares/solver/expressions/nodes/NegationNode.h"
+#include "antares/solver/expressions/nodes/ParameterNode.h"
+#include "antares/solver/expressions/nodes/PortFieldNode.h"
+#include "antares/solver/expressions/nodes/PortFieldSumNode.h"
+#include "antares/solver/expressions/nodes/SubtractionNode.h"
+#include "antares/solver/expressions/nodes/SumNode.h"
+#include "antares/solver/expressions/nodes/VariableNode.h"
```

Comment:
@a-zakir do you know why we can't compile this class that uses EvalVisitor, without adding all these includes ?